AUTHORS

```diff
@@ -4,7 +4,11 @@ those who explicitly didn't want to be mentioned. People with a * next
 to their names are not found in the metadata of the git history. This
 file is generated automatically by running `./bin/authors_update.py`.
 
+<<<<<<< HEAD
 There are a total of 1091 authors.
+=======
+There are a total of 1097 authors.
+>>>>>>> cf3e8ea465 (commit to 'master')
```

Comment:
When resolving a merge conflict you need to delete this extra stuff that git puts in.